packages/tao/src/commands/run.ts

```diff
@@ -263,19 +263,32 @@ async function runExecutorInternal<T extends { success: boolean }>(
       );
     }
   } else {
+    let errors = [];
```

Comment:
I think this can be undefined to begin. We don't need to collect any errors in an array. We only need to print the last error (if it's there).